package-lock.json

```diff
@@ -1382,9 +1382,9 @@
       "dev": true
     },
     "ajv": {
-      "version": "6.10.0",
-      "resolved": "https://registry.npmjs.org/ajv/-/ajv-6.10.0.tgz",
-      "integrity": "sha512-nffhOpkymDECQyR0mnsUtoCE8RlX38G0rYP+wgLWFyZuUyuuojSSvi/+euOiQBIn63whYwYVIIH1TvE3tu4OEg==",
+      "version": "6.10.2",
+      "resolved": "https://registry.npmjs.org/ajv/-/ajv-6.10.2.tgz",
+      "integrity": "sha512-TXtUUEYHuaTEbLZWIKUr5pmBuhDLy+8KYtPYdcV8qC+pOZL+NKqYwvWSRrVXHn+ZmRRAu8vJTAznH7Oag6RVRw==",
```

Comment:
Can/Should I add it to the .gitignore?

Reply:
I think you can and I think you probably should (at least for now, let me know if you find something that contradicts that)

Reply:
[The npm docs](https://docs.npmjs.com/files/package-lock.json) and a few other articles say it should be committed because it describes the dependency tree and enables us to easily go through old commits.

Reply:
:+1: